modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -26,7 +26,11 @@ kubectl describe nodes
 
 == Limitations
 
-Redpanda does not support decreasing the CPU cores for brokers in an existing cluster.
+Redpanda supports decreasing the CPU cores for brokers in an existing cluster as of version 24.2.
+
+In v24.2, you must enable the `node_local_core_assignment` flag to be able to decrease the core count.
```

Comment:
Can you add a note that decreasing CPU core count is not supported for production clusters?